.github/workflows/maven.yml

```diff
@@ -0,0 +1,211 @@
+name: Java CI with Maven and native builds
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  # Java version to use for the release
+  RELEASE_JAVA_VERSION: 11
+  UBUNTU_DIST: focal
+  UBUNTU_MIRROR_AMD64: http://azure.archive.ubuntu.com/ubuntu
```

Comment:
Why the different mirror?

Reply:
Because Azure only mirrors amd64, not the other architectures. Which is very unfortunate and complicates everything :(